css/team.css

```diff
@@ -0,0 +1,47 @@
+.team--grey {
+  background-color: #f5f4fa;
```

Comment:
this can be also saved in css variables, if this color is used in more than 1 place in the project